tests/datasets/test_cv4a_kenya_crop_type.py

```diff
@@ -113,3 +119,17 @@ def test_invalid_bands(self) -> None:
 
         with pytest.raises(ValueError, match="is an invalid band name."):
             CV4AKenyaCropType(bands=("foo", "bar"))
+
+    def test_plot(self, dataset: CV4AKenyaCropType) -> None:
```

Comment:
This test is a bit hacky. Instead of parametrizing the fixture, why not have two plotting tests, one that takes the fixture as input and plots correctly, and another that creates a new dataset with a single band and raises an error?

Reply:
Okay, I got the code from seco.py, where the bands were tested like this. With your suggestion, I am actually not sure how to create a new dataset in another test method, because the dataset_id, tile_names etc are fixed and I cannot instantiate the dataset with the fake data. Or do you mean that I should create a second new fixture, where a dataset with just one band is created?

Reply:
You can use the same dataset fixture in your test method but create a new dataset with the same root/dataset_id/tile_names/etc. but with a different set of bands.